tests/test_sca.py

```diff
@@ -33,7 +33,16 @@ def test_parse_components():
     # directives in nested go.mod files
     assert "../" not in names
     assert "github.com/Microsoft/go-winio" in names
-    assert results[0]["analysis_meta"] == {"source": "syft", "version": "0.48.1"}
+    assert results[0]["analysis_meta"] == {"source": "syft-0.60.1"}
+
+
+def test_parse_maven_components():
+    with open("tests/data/hawkular-metrics-schema-installer-syft.json", "r") as maven_test_data:
+        results = Syft.parse_components(maven_test_data.read())
+    assert len(results) > 0
+    group_ids = [r["meta"].get("group_id") for r in results if r["type"] == Component.Type.MAVEN]
+    assert "" not in group_ids
```

Comment:
I think you're trying to assert that .get() always returns a value, right? But by default .get() returns None if a key isn't found. So either change the first line to ```.get("group_id", "")``` or the second line to ```assert None not in group_ids```

Reply:
You are correct, updated.